docs/src/components/original-documentation/index.js

```diff
@@ -0,0 +1,18 @@
+// Dependencies
+import {h} from 'preact';
+
+// Class
+export default ({children, ...props}) => {
+  let child = children && children[0];
+  let orgDocUrl = `https://material.io/develop/web/components/${props.link}`;
+
+  return (
+    <div className="mdc-typography--display1">
+      Original documentation
+      <div className="mdc-typography--body">
+        This component encapsulates <span className="strong">{child}</span>
+        , you can refer to its documentation <a href={orgDocUrl}>here</a>.
+      </div>
+    </div>
```

Comment:
Shouldn't is be like
``` typescript tsx
    <div className="mdc-typography--display1">
      Original documentation
    </div>
    <div className="mdc-typography--body">
      This component encapsulates <span className="strong">{child}</span>
      , you can refer to its documentation <a href={orgDocUrl}>here</a>.
    </div>
```

Reply:
as far as i can see effect in rendering is identical, and there is an encapsulating div needed in return right? Adding a new div around both results loses the white line above the header, thats why i used this way.

Reply:
npm run tslint:fix is happy locally here :-( Could there be a version difference somewhere?

Reply:
i just thought to start a little cleanup ;-) haha.  not sure if that goal is still met...

Reply:
Oh yes right the docs aren't tslint because they are js. 
Try `prettier:fix`

Reply:
That would also be okay I think:
```
    <div className="mdc-typography--body">
      <div className="mdc-typography--display1">
        Original documentation
      </div>
      This component encapsulates <span className="strong">{child}</span>
      , you can refer to its documentation <a href={orgDocUrl}>here</a>.
    </div>
```
But how do you want to explain some thing like a screen reader that you put the body into the heading, that makes no sense.

Reply:
Sorry, I'm used to just hit a combo in my IDE and it's okay and the rest is catched by the prettier CI but that doesn't help with a PR...

Try running `prettier` manually (with write) via `npx`

Reply:
If I remember correctly prettier is used by eslint as a plugin, so that could work either

Reply:
thanks. prettier issue found, div changed resultated in more compact sample code. That was not intended but is ok I think. Getting somewhere now.